services/gtfs-rt-archiver-v3/gtfs_rt_archiver_v3/consumer.py

```diff
@@ -12,14 +13,35 @@
 
 from huey.consumer_options import ConsumerConfig
 from prometheus_client import start_http_server
+from requests import HTTPError
 
 from .tasks import huey, load_secrets
 
 
+def set_exception_fingerprint(event, hint):
+    if "exc_info" not in hint:
+        return event
+
+    exception = hint["exc_info"][1]
+    if isinstance(exception, HTTPError):
+        event["fingerprint"] = [
+            "{{ default }}",
+            str(exception),
+            str(exception.response.status_code),
+        ]
+
+    return event
+
+
 def main(
     port: int = os.getenv("CONSUMER_PROMETHEUS_PORT", 9102),
     load_env_secrets: bool = False,
 ):
+    sentry_sdk.init(
+        environment=os.getenv("AIRFLOW_ENV"),
+        # TODO: add me back if fetch errors are being grouped together too much
```

Comment:
per offline convo, should add this back in -- `ConnectionError` in particular is too generic and not actionable without URL

Reply:
Can do.

Reply:
This is done; it will already work for dbt because a custom exception is constructed with the unique_id and message, and the schedule download failures come from exception logs rather than unhandled exceptions (which we do want to maintain for RT so huey tasks exit with a failure).